uint8 NO_NAVIGATION = 0
uint8 VEL_FLIGHT_MODE_COMMAND = 1
uint8 POS_FLIGHT_MODE_COMMAND = 2


Header header

uint8 command_mode
float32 target_pos_x
float32 target_vel_x

float32 target_pos_y
float32 target_vel_y

float32 target_vel_psi
float32 target_psi

uint8 pos_z_navi_mode

float32 target_pos_z
float32 target_pos_diff_z

